google/ggrc-core
# GGRC-6068 Add 4 new attributes for Risk

# Issue description

Add 4 new attributes for risk objects:
- risk type,
- threat source,
- threat event,
- vulnerability

# Steps to test the changes

1. Be able to create Risk object with new attributes.
2. (propose changes) User should be able to propose updates for these attributes.
3. (restore version) User should be able restore the value of this attributes. 
4. (object review) updates for these fields should revert object to 'Unreviewed' state.
5. (change log) any updated to these attributes should be tracked in change log of a risk.
6. (import) user should be able to update values to these attributes via import.
7. (export) user should be able to export these values.
8. (filtering) user should be able to filter a list of risks by these attributes.
9. (advanced search) user should be able to search risks by these attributes.
10. (tree view) user should be able to this columns in tree view (by default should be hidden).

# Solution description
Added new attributes fields to Risk model and created migration to add new columns.

# Sanity checklist


# Migration checklist

# PR Review checklist
